tools/s2i-dpdk/Dockerfile

```diff
@@ -0,0 +1,78 @@
+# dpdk-centos7
+FROM openshift/base-centos7
+#FROM centos:7
+
+LABEL maintainer="Sebastian Scheinkman <sebassch@gmail.com>"
```

Comment:
Does it make sense to have your rh address instead of gmail one?

Reply:
I prefer to have this one here. This is the one I am using for github also